### ISSUE TITLE
PHP-CS-Fixer 3.47.0 changes class name `Assert` to `assert`

### ISSUE THREAD

**Issue by sebastianbergmann** [PHP-CS-Fixer/PHP-CS-Fixer#7747]
Using https://github.com/sebastianbergmann/phpunit/blob/8.5/.php-cs-fixer.dist.php on the `8.5` branch of PHPUnit, the following unwanted change is made:

```patch
diff --git a/src/Framework/TestCase.php b/src/Framework/TestCase.php
index a0edaf12ff..8349341e77 100644
--- a/src/Framework/TestCase.php
+++ b/src/Framework/TestCase.php
@@ -114,7 +114,7 @@
 use Text_Template;
 use Throwable;
 
-abstract class TestCase extends Assert implements SelfDescribing, Test
+abstract class TestCase extends assert implements SelfDescribing, Test
 {
     private const LOCALE_CATEGORIES = [LC_ALL, LC_COLLATE, LC_CTYPE, LC_MONETARY, LC_NUMERIC, LC_TIME];
```

**Comment by Wirone** [comment]
It's because of [`use function assert;`](https://github.com/sebastianbergmann/phpunit/blob/c7a9a16a6a90bba560fedf604f19da42073959f4/src/Framework/TestCase.php#L25) and will be fixed in either #7704 or #7705. Basically it's the same problem as in #7744.

**Comment by Wirone** [comment]
Fixed in #7750.
